src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/base-step/base-step.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import {
+    FormBuilder,
+    FormControl,
+    FormGroupDirective,
+    ReactiveFormsModule,
+} from "@angular/forms";
+import { BaseStepComponent } from "./base-step.component";
+import { FetchKind } from "../../add-polling-source-form.types";
+import { FETCH_FORM_DATA } from "../data/fetch-form-data";
+import { FETCH_STEP_RADIO_CONTROLS } from "../../form-control.source";
+import { SetPollingSourceSection } from "src/app/shared/shared.types";
+import { TooltipIconComponent } from "src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component";
+import { NgbTooltipModule } from "@ng-bootstrap/ng-bootstrap";
+
+describe("BaseStepComponent", () => {
+    let component: BaseStepComponent;
+    let fixture: ComponentFixture<BaseStepComponent>;
+    const fb = new FormBuilder();
+    const formGroupDirective = new FormGroupDirective([], []);
+    formGroupDirective.form = fb.group({
+        fetch: fb.group({
+            kind: new FormControl("url"),
+        }),
+    });
```

Comment:
These fields are not needed anywhere, except the `beforeEach` block. Make them local.

Reply:
Fixed.